python/lsst/daf/butler/configs/datastores/formatters.yaml

```diff
@@ -77,3 +77,4 @@ AstropyQTable: lsst.daf.butler.formatters.astropyTable.AstropyTableFormatter
 ExtendedPsf: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
 HealSparseMap: lsst.pipe.tasks.healSparseMapping.HealSparseMapFormatter
 ButlerLogRecords: lsst.daf.butler.formatters.logs.ButlerLogRecordsFormatter
+TaskMetadata: lsst.daf.butler.formatters.json.JsonFormatter
```

Comment:
Wait, doesn't the change from YAML to JSON mean that older metadata cannot be read into the new class?

Reply:
No. The formatter is burned in on put so anything written with the YAML formatter will be read by the YAML formatter. Those are a different storage class anyhow (PropertySet -- that's the problem that makes switching hard).